webapp/components/wpt-results.html

```diff
@@ -151,6 +151,10 @@
   <script>
     /* global TestRunsBase */
     class WPTResults extends TestRunsBase {
+      static get UrlToResultsPath() {
```

Comment:
lowerCamelCase as in rest of file, or is there a convention here I'm missing?

Reply:
Done.